### ISSUE TITLE
Cleanup and documentation

### ISSUE THREAD

**Issue by adelespinasse** [springml/case_routing#2]
This code is hard to follow. A high-level explanation in the README.md would be a big help, as would more docstrings, and I think there's a lot of dead code (I noticed duplicate code among several Python source files in the top-level directory, for example).

Maybe this repo wasn't really intended as a clear demonstration of an email classifier, but [this blog post](https://cloudplatform.googleblog.com/2017/08/intelligent-email-categorization-with-machine-learning.html) implied that it is.

**Comment by ghost** [comment]
Here's a more detailed write-up that covers the modeling aspects as well.
https://docs.google.com/document/d/1km1tkU-KyHderCeR-qCWV1uRd_D3JwoC1s4ZAIw2kXQ/edit

Hope this helps.

**Comment by adelespinasse** [comment]
Thanks... that document is private. I just requested access, but if it's the best writeup, maybe it should be world-readable?

**Comment by moyinho20** [comment]
Hi! I just requested for access to this document. Please grant :)

**Comment by adelespinasse** [comment]
I was never given access either. I think this project is dead, and the document was probably never intended for the public.
